src/gui/XournalppCursor.cpp

```diff
@@ -254,8 +275,10 @@ void XournalppCursor::updateCursor()
 		{
 			gdk_window_set_cursor(gtk_widget_get_window(xournal->getWidget()), cursor);
 		}
-
-		gtk_widget_set_sensitive(xournal->getWidget(), !this->busy);
+		else if(useDefault)
```

Comment:
`useDefault` should be renamed, since it does not "use a default cursor" but rather disables inputs.